test/raven.client.js

```diff
@@ -359,7 +340,7 @@ describe('raven.Client', function() {
           foo: null
         }
       };
-      kwargs.extra.foo = kwargs;
+      kwargs.extra.foo = kwargs.extra;
```

Comment:
It had to be changed, as referencing `kwargs` directly wouldn't work in this test, as it's not the same reference anymore.